css/port.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.container {
+  max-width: 375px;
+  height: 100vh;
+  margin: auto;
+  background-image: url('../images/header-shapes\ mobile.png');
+}
+
+.header {
+  padding: 0 35px;
+}
```

Comment:
- [ ] [OPTIONAL] I would suggest to remove this padding to make it more identical to the figma design